public/css/toggle.css

```diff
@@ -0,0 +1,183 @@
+.location-pin {
+    position: absolute;
+    display: block;
+    float: left;
+}
+
+.location-pin:hover {
+    cursor: ;
+}
+
+.toggleButton {
+  bottom: 5%;
+  left: 52%;
+  transform: translateX(-140%);
+  position: absolute;
+}
+
+
+@media (max-width:1300px) {
+  .toggleButton {
+      transform: translateX(-135%);
+  }
+}
+
+@media (max-width:1200px) {
+  .toggleButton {
+      transform: translateX(-130%);
+  }
+}
+
+@media (max-width:1000px) {
+  .toggleButton {
+      transform: translateX(-100%);
+  }
+}
+
+@media (max-width:800px) {
+  .toggleButton {
+      transform: translateX(-95%);
+  }
+}
+
+@media (max-width:600px) {
+  .toggleButton {
+      transform: translateX(-77%);
+  }
+}
+
+@media (max-width:400px) {
+  .toggleButton {
+      transform: translateX(-90%);
+  }
+}
+
+.event-card {
+  transform: translate(110%, 30%);
+}
+
+.event-card h2 {
+    color: white;
+}
+
+.more-info {
+  transform: translate(5%, -59%);
+  width: 90%;
+}
+
+.additional {
+  transform: translateY(-8%)
+}
+
+.btn {
+  position: relative;
+  display: inline-block;
+  margin: 1em;
+  padding: 1em 2em;
+  border: 0;
+  border-radius: 30px;
+  text-align: center;
+  white-space: nowrap;
+  cursor: pointer;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px;
+  background-color: #42626d;
+}
+
+.btn:hover {
+  box-shadow: inset 0 -3px 0 rgba(0, 0, 0, 0.5);
+}
+
+.btn:focus {
```

Comment:
You shouldn't use `focus` to give the buttons the highlight effect because
* The floor 1 button isn't highlighted by default
* When you click something other than the button, the button gets unhighlighted

Instead, there should be a selector for `.btn.selected` or something, and clicking a button adds the `.selected` class to that button while removing it from other buttons